core/src/main/java/com/klaytn/caver/wallet/KlayWalletUtils.java

```diff
@@ -200,7 +195,7 @@ public static boolean isValidAddress(String input) {
             return false;
         }
 
-        return cleanInput.length() == ADDRESS_LENGTH_IN_HEX;
+        return (cleanInput.length() <= ADDRESS_HEX_SIZE && cleanInput.matches("^[0-9A-Fa-f]+$"));
```

Comment:
Inside of `String.matches()` java use `Pattern` instance which is quite expensive object to create. If you create this `Pattern` instance as 'static' variable and reuse it, you could increase performance better. You can use the regex like below.
```
    private static final Pattern ADDRESS = Pattern.compile("^[0-9A-Fa-f]+$");
    
    static boolean isAddress(String s) {
        return ADDRESS.matcher(s).matches();
    }
```